pulseapi/profiles/urls.py

```diff
@@ -20,4 +21,9 @@
     # note that there is also a /myprofile route
     # defined in the root urls.py which connects
     # to the UserProfileAPIView class.
+    url(
```

Comment:
nit: can this be a simple `r'^$'` route that uses a list api view defined (mainly don't name it 'search' since it's a view to list profiles?)

Reply:
I tried that, but that makes it go into an infinite redirect loop. If you know how to make it not do that, though, an empty route rather than `search` would be much better.

Reply:
let's land the pr this is dependent on first and then we can try to debug that together.

Reply:
landed.